libs/wingc/src/capture.rs

```diff
@@ -240,6 +239,9 @@ fn scan_captures_in_expression(
 					}
 				}
 			}
+			Reference::TypeProperty { .. } => {
+				// TODO: handle access to static preflight memebers from inflight
```

Comment:
Issue reference please 

Also- should we emit a diagnostic here?

Reply:
@yoav-steinberg opened issue #1669.
After an offline chat with @yoav-steinberg I understand emitting the right diagnostic here calls for some refactoring. I suggest we merge the PR as is and since #1669 is p1 we should address it before beta.